test/xpu/fin_grain/run_fine_grain.py

```diff
@@ -22,11 +22,15 @@
     "test_compare_cpu__refs_rsub_xpu_bfloat16",
     "test_compare_cpu_add_xpu_bfloat16",
     "test_compare_cpu_sub_xpu_bfloat16",
+    "test_compare_cpu_acos_xpu_complex128",
```

Comment:
What's the failure?

Reply:
When the input is complex(inf,inf)，the output of cpu is nan+nanj，the output of xpu and cuda is 0.7854-infj

Reply:
can be tested in test_ops_xpu.py::TestCommonXPU::test_python_ref__refs_acos_xpu_complex128 PASSED